### ISSUE TITLE
add ml-analysis argo workflows to automated deployment

### ISSUE THREAD

**Issue by durandom** [operate-first/continuous-deployment#38]
maybe use https://github.com/aicoe-aiops/fedora-mailing-list-analysis

**Comment by tumido** [comment]
@cdolfi Is that an active repo? Seems a bit empty. Also Fedora doesn't use Mailman - it uses HyperKitty instead, the web scrapper would have to be redone.

**Comment by cdolfi** [comment]
Yes it is. I will be adding in files this week. I had some delays in the middle of moving my work over to Github. I have already scrapped and cleaned all of the data, but I will look into this issue and see if I should go back and redo this step.
